src/applications/static-pages/ivc-champva/10-7959f-1/App.js

```diff
@@ -26,21 +26,23 @@ const App = ({ formEnabled }) => {
         <h3>
           <strong>Option 1: Online</strong>
         </h3>
-        <a
-          className="vads-c-action-link--blue"
-          href="/health-care/foreign-medical-program/register-form-10-7959f-1"
-        >
+        <a href="/health-care/foreign-medical-program/register-form-10-7959f-1">
```

Comment:
Just an fyi that the [`va-link` web component]( https://design.va.gov/storybook/?path=/docs/components-va-link--docs) is available. 

And if at some point you need the action link again, there is a [`va-action-link` web component](https://design.va.gov/storybook/?path=/docs/components-va-link-action--docs) for that now too.